File: leetcode/JavaScript实现/1-99题/1-19题/0009.js

// 自己的实现，可能有点辣鸡，到时候改进
var isPalindrome = function(x) {
    let s = x.toString();
    let len = s.length;
    for(let i = 0; i < len / 2; ++i) {
        if(s[i] != s[len - i -1])
            return false;
    }
    return true;
};

File: leetcode/C++实现/1-99题/0009.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        string s=to_string(x);
        int s_size=s.size();
        for(int i=0;i<s_size/2;++i)
        {
            if(s[i]!=s[s_size-1-i])
            {
                return false;
            }
        }
        return true;
    }
};
